content/en/docs/started/installing-kubeflow.md

```diff
@@ -200,12 +334,16 @@ The following table lists distributions which are <em>maintained</em> by their r
   </table>
 </div>
 
-## Raw Kubeflow Manifests
+### Raw Manifests
 
-The raw Kubeflow Manifests are aggregated by the [Manifests Working Group](https://github.com/kubeflow/community/tree/master/wg-manifests)
-and are intended to be used as the **base of packaged distributions**.
+The raw Kubeflow Manifests are aggregated by the Manifests Working Group and are intended to be
+used as the **base of packaged distributions**.
 
-Advanced users may choose to install the manifests for a specific Kubeflow version by following the
+Kubeflow Manifests contain all Kubeflow Components, Kubeflow Central Dashboard, and other Kubeflow
+applications which makes **Kubeflow Platform**. This installation is helpful when you want to try
+out the end-to-end Kubeflow Platform capabilities.
+
+Users may choose to install the manifests for a specific Kubeflow version by following the
 instructions in the `README` of the [`kubeflow/manifests`](https://github.com/kubeflow/manifests) repository.
 
 - [**Kubeflow 1.8:**](/docs/releases/kubeflow-1.8/)
```

Comment:
```
- [**Kubeflow 1.8:**](/docs/releases/kubeflow-1.8/)
  - [`v1.8-branch`](https://github.com/kubeflow/manifests/tree/v1.8-branch#installation) <sup>(development branch)</sup>
  - [`v1.8.0`](https://github.com/kubeflow/manifests/tree/v1.8.0#installation)
- [**Kubeflow 1.9:**](/docs/releases/kubeflow-1.9/)
  - [`v1.9-branch`](https://github.com/kubeflow/manifests/tree/v1.9-branch#installation) <sup>(development branch)</sup>
  - [`v1.9.0`](https://github.com/kubeflow/manifests/tree/v1.9.0#installation)
```

I think we should either reference the master or 1.9 branch, but 1.7 is not supported anymore.

Reply:
@juliusvonkohout Can we update it once we release Kubeflow 1.9 ?

Reply:
We should not link to unreleased versions.